org.uqbar.project.wollok/src/org/uqbar/project/wollok/messages.properties

```diff
@@ -12,7 +12,7 @@ WollokDslValidator_BAD_USAGE_OF_IF_AS_BOOLEAN_EXPRESSION = Bad usage of if ! You
 WollokDslValidator_BINARYOP_ONLY_ON_VARS = can only be applied to variable references
 WollokDslValidator_CANNOT_ASSIGN_TO_ITSELF = Cannot assign a variable to itself. It does not have any effect
 WollokDslValidator_CANNOT_INSTANTIATE_ABSTRACT_CLASS = Cannot instantiate abstract class
-WollokDslValidator_CANNOT_MODIFY_VAL = Cannot modify values
+WollokDslValidator_CANNOT_MODIFY_VAL = Cannot modify constants
```

Comment:
Probably you also want to change the spanish version. I think that is a similar file name plus "es" or something